app/db/schemas/schemaV5.js

```diff
@@ -0,0 +1,28 @@
+import ProgramLanguage from '../migrations/v3/programLanguage';
+import Scorecard from '../migrations/v5/scorecard';
+import schemaHelper from '../../helpers/schema_helper';
+import { schemaNames } from '../../constants/schema_constant';
+
+import uuidv4 from '../../utils/uuidv4';
+
+const changedSchemas = [
+  { label: schemaNames[0], data: Scorecard },
+  { label: schemaNames[13], data: ProgramLanguage },
+];
+
+const schemaV5 = {
+  schema: schemaHelper.getSchemas(changedSchemas),
+  schemaVersion: 5,
+  migration: (oldRealm, newRealm) => {
+    if (oldRealm.schemaVersion < 5) {
+      const oldObjects = oldRealm.objects('Scorecard');
+      const newObjects = newRealm.objects('Scorecard');
+
+      for (let i = 0; i < oldObjects.length; i++) {
+        newObjects[i].uuid_on_app = !oldObjects[i].uuid_on_app ? uuidv4() : oldObjects[i].uuid_on_app;
```

Comment:
@limkimsan what is the purpose of introducing uuid_on_app?

Reply:
The uuid_on_app is for sending to scorecard milestone API, and it is used for defined different records of the scorecard in different devices.

Reply:
Bong, do not merge this PR. I need to some changes on this PR.

Reply:
Bong, I have finished updating the PR.

Reply:
in term of sending uuid_on_app, please send the device_id instead if so

Reply:
The current data send to milestone data includes both device_id and uuid_on_app. I will check with bong Sokly if we can use device_id only.